pwiz_tools/Skyline/Model/RetentionTimes/KdeAligner.cs

```diff
@@ -73,56 +76,95 @@ public KdeAligner(int origXFileIndex, int origYFileIndex, int resolution = 1000)
             _resolution = resolution;
         }
 
-        public KdeAligner(int resolution = 1000) 
+        /// <summary>
+        /// Constructs a KdeAligner with a specific resolution and stretchFactor
+        /// </summary>
+        /// <param name="resolution">The number of points in the X and Y axes </param>
+        /// <param name="eccentricity">Experimental parameter controlling the amount that the CosineGaussian stamp is stretched.
+        /// This value is interpreted as the eccentricity of an ellipse whose major axis is along the Y=X axis.</param>
+        public KdeAligner(int resolution = 1000, double eccentricity = 0)

```

Comment:
This "eccentricity" parameter should be removed. It was a misguided thing that I thought might help some scenario but never improved things.

Reply:
Will do, thanks